### PR TITLE
chaning button edges, spacing and truncate using words

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by smarzban** [comment]
👍

File: freebasics/static/css/template-base.css [hunk 1]
```diff
@@ -210,16 +210,25 @@ h6{
...
 	cursor:pointer;
-	color:#fff;
 	font-size: 12px;
 	padding: 8px 22px;
 	text-decoration:none;
 	border: 15px;
   text-align: center;
 }
+.flatbutton-input:hover {
+	background-color:#c736c0;
```
Comment (miltontony) [inline freebasics/static/css/template-base.css:226]: Why are you hardcoding this purple button. I thought we were going to use `accent 1`

**Comment by codiebeulaine** [comment]
@saeedmb8 ready for review again

**Comment by smarzban** [comment]
👍

**Comment by smarzban** [comment]
@codiebeulaine Please make separate PRs for different changes. 👍

**Comment by codiebeulaine** [comment]
@saeedmb8 yeah sorry about that

**Pull request merged**